### DISCUSSION TITLE
Add a Taxon Identifiers table

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5732] [Feature Requests]
Similar to Agent Identifiers

**Comment by ianengelbrecht** [comment]
After suggesting this Grant and some more thought, perhaps this needs more deliberation. Superficially this is the same as having agent identifiers. Arguably, taxon identifiers are more important than agent identifiers (because it's more important to be unequivocally sure what taxon a record refers to than to be unequivocally sure who collected it or identified it, but that's just IMO). The challenge is that we don't have a similarly well resolved system for minting and allocating taxon identifiers like we do for people (ORCID, wikidata, etc), but we can use Zoobank, COL, etc. A significant complication is whether the identifiers refer to taxon names or taxon concepts. So it may not be possible to implement this in Specify until the name/concept issue is better developed (we had some discussion about this a while ago in the TAC). It would be a horrendous mess if users treat their taxon records in Specify as names, and dump all identifiers for all concepts related to a name into the name record. That would defeat the purpose of having identifiers for taxa :-/

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8209
